packages/amplify_auth_cognito/example/ios/unit_tests/amplify_auth_cognito_tests.swift

```diff
@@ -1344,7 +1344,7 @@ class amplify_auth_cognito_tests: XCTestCase {
                 XCTAssertEqual(false, res.toJSON()["isSignedIn"] as? Bool)
                 XCTAssertEqual("testid", res.toJSON()["identityId"] as? String)
                 // userSub error will result in map with one 'error' key
-                XCTAssertEqual(1, (res.toJSON()["tokens"] as?  [String: String])!.count)
+                XCTAssertEqual(nil, (res.toJSON()["tokens"] as?  [String: String]))
```

Comment:
Nit: Use `XCTAssertNil` instead?
```suggestion
                XCTAssertNil(res.toJSON()["tokens"] as? [String: String])
```

Reply:
Done!